.github/workflows/build-and-release-dev.yml

```diff
@@ -0,0 +1,151 @@
+name: Build a dev version for all packages and release them
+
+on:
+  pull_request_review:
+    types: [submitted]
```

Comment:
I don't get why we want to release a dev version when a review on a PR is submitted. 
Don't we want it on a PR merge, and on manual dispatch?

Reply:
This is just for testing purposes. In the final version this will be a manual workflow